.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Release zip package
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  release:
+    name: Release zip package
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.x
+
+    - name: Cache dotnet packages
+      uses: actions/cache@v2.1.5
+      with:
+        path: |
+          */obj/project.assets.json
+          */obj/*.csproj.nuget.*
+          .nuget
+        key: dotnet-packages-cache
+
+    - name: Restore dependencies
+      run: dotnet restore --packages .nuget
+
+    - name: Build solution
+      run: dotnet publish --no-restore -c Release -o build/out/
+
+    - name: Create zip
+      run: zip -q -r friends-and-places-server.zip build/out/
```

Comment:
Zip ist kein default tool von Ubuntu, bist du sicher dass das in der Action verfügbar ist? Ggf. tar -czf benutzen

Reply:
Berechtigter Einwand.
zip ist auf ubuntu-latest Maschinen verfügbar:
https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md
(ganz unten, vorletzter Eintrag)